src/PHPDocker/Template/nginx.conf.twig

```diff
@@ -13,7 +13,7 @@ server {
             {% set indexFile = 'index' %}
         {% endif %}
 
-    root /application/web;
+    root {{ dockerWorkingDir }}/web;
```

Comment:
See comment above on the controller about form values with trailing slashes, they could potentially cause issues here and below in `/public`